etabotsite/default_settings.json

```diff
@@ -0,0 +1,45 @@
+{
+    "LOCAL_MODE": true,
```

Comment:
Please add default log_filename_with_path to a local file within the home directory.

Reply:
done, please review and resolve if ok